services/ui-src/src/page/chip-spa/CHIPSPARAIForm.tsx

```diff
@@ -16,7 +16,8 @@ export const chipSPARAIFormInfo: OneMACFormConfig = {
 
 const CHIPSPARAIForm: FC = () => {
   const location = useLocation<FormLocationState>();
-  chipSPARAIFormInfo.landingPage += `/${location.state?.componentId}`;
+  if (location.state?.componentId)
+    chipSPARAIFormInfo.landingPage = `${ONEMAC_ROUTES.CHIP_SPA_DETAIL}/${location.state?.componentId}`;
```

Comment:
ooohh.....I like it! Also nice job tracking down this issue. It definitely would have taken me far longer to realize this was the cause.

Reply:
Well.... when one is the cause, it is often easier to see the mess!